src/components/CytoscapeLayout/CytoscapeLayout.tsx

```diff
@@ -79,4 +77,17 @@ export default class CytoscapeLayout extends React.Component<CytoscapeLayoutProp
       </div>
     );
   }
+
+  private updateGraphElements(newNamespace: string) {
+    API.GetGraphElements(newNamespace, null)
+      .then(response => {
+        const elements: { [key: string]: any } = response['data'];
+        console.log(elements);
+        this.setState(elements);
+      })
+      .catch(error => {
+        this.setState({});
```

Comment:
Shouldn't this be `this.setState([])`  because of: https://github.com/swift-sunshine/swsui/pull/73/files#diff-f3fe4c447ae60d0c289d35c2cc7bfcfcR24 ?

Reply:
I copied and pasted that chunk of code as is... I'm not sure what's doing the empty object... Perhaps it should be `this.setState({ elements: [] });`

Reply:
With type safety it shouldn't even be an issue. That is what `CytoscapeLayoutState` is for.